### PR TITLE
Display match result from match buyer command

### PR DESCRIPTION
**Pull request by zengzihui**
- Create MatchResultListCard.fxml and MatchResultListPanel.fxml to display the match results from match buyer command, only the matched seller with its matched houses are displayed

### CODE REVIEW

**Review by KhoonSun47 — APPROVED** [review]
LGTM! Good job in implementing this!

**Review by lokidoki102 — APPROVED** [review]
LGTM 👍👍

**Review by felixchanyy — APPROVED** [review]
👍

File: src/main/resources/view/CustomisedTheme.css [hunk 1]
```diff
@@ -19,3 +19,9 @@
 .details-card {
 
 }
+
+.test-3 {
+    -fx-background-color: derive(#D53B1A, 100%);
+    -fx-border-color: derive(#D53B1A, 100%);
+    -fx-border-top-width: 1px;
+}
```
Comment (felixchanyy) [inline src/main/resources/view/CustomisedTheme.css:27]: No newline at EOF

File: src/main/java/seedu/address/logic/commands/MatchBuyerCommand.java [hunk 2]
```diff
@@ -45,10 +47,19 @@ public CommandResult execute(Model model) {
...
             PriceAndHousingTypePredicate predicate = new PriceAndHousingTypePredicate(budget.toPrice(), housingType);
+
             model.updateFilteredSellerList(predicate);
 
-            ObservableList<House> filteredSellerList = model.getFilteredSellerList(predicate);
+            ObservableList<House> filteredSellerList = model.getAllFilteredHouseList(predicate);
+            model.setState(State.MATCH_RESULTS);
+            model.showMatchResults(model.getFilteredSellerList());
+
+            for (Seller filteredSeller : model.getFilteredSellerList()) {
```
Comment (felixchanyy) [inline src/main/java/seedu/address/logic/commands/MatchBuyerCommand.java:57]: Should this be removed?

**Pull request merged**